src/java.base/share/classes/java/text/DecimalFormat.java

```diff
@@ -2438,7 +2479,7 @@ int subparseNumber(String text, int position,
                        boolean isExponent, boolean[] status) {
         // process digits or Inf, find decimal position
         status[STATUS_INFINITE] = false;
-        if (!isExponent && text.regionMatches(position,symbols.getInfinity(),0,
+        if (!isExponent && text.regionMatches(position, symbols.getInfinity(),0,
```

Comment:
```suggestion
        if (!isExponent && text.regionMatches(position, symbols.getInfinity(), 0,
```

Reply:
Thanks, this and the other suggestions you provided should be fixed.